main.py

```diff
@@ -1,8 +1,13 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Depends
 from event_feed import EventFeed
 from database import engine, Base, get_db
 from models import Warship, Player
 from entities import Orientation
+from sqlalchemy.orm import Session
+import models, schemas
```

Comment:
w linii 4 importujemy bezpośrednio dwa modele z pliku models.py, tutaj importujemy cały moduł, należałoby uporządkować te importy, zdecydować się na jedno albo drugie, tj. albo `import models, schemas` i do każdej zmiennej dostajemy się przez models.Player, models.Warship, schemas.PlayerBase, albo `from models import Player` i wtedy analogicznie, mamy dostęp do samego Player

tak długo jak te nazwy ze sobą nie kolidują - odpowiednik Player'a to PlayerBase etc. nie ma potrzeby dostawać się do tego w sposób models.Player, sugeruję drugi sposób i proszę, aby dostosować kod pod te zmiany - uporządkować importy i wszędzie w kodzie podmienić jak należy